mgmtd/mgmt_frntnd_adapter.h

```diff
@@ -0,0 +1,200 @@
+/*
+ * MGMTD Frontend Client Connection Adapter
+ * Copyright (C) 2021  Vmware, Inc.
+ *		       Pushpasis Sarkar <spushpasis@vmware.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; see the file COPYING; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#ifndef _FRR_MGMTD_FRNTND_ADAPTER_H_
+#define _FRR_MGMTD_FRNTND_ADAPTER_H_
```

Comment:
The challenge will be now to modify the original commits. Is it okay to do this change in a 9th commit. These abbreviations are spread out across 5 different commis. It will be hard to modify the individual commits using 'git rebase -I'. But definitely can do the abbreviations replacement in a separate commit. Will that be okay with you? Hope anyone else will not have a problem too.

Reply:
BTW before you do this, what are we going with? Might as well argue about the color of the bikeshed during the local/remote valiadtion meeting too.. I'm with @idryzhov I'd like to see "fe" and "be" used. There's even precedent.. If you google "frontend backend abbreviation" the one that pops up is "febe". FWIW it's a good idea to do a google search anytime you pick an abbreviation, that way you are more likely to not be dreaming up your own when a standard exists (e.g., transaction => txn came up for me)..